tests/test_splunk_image.py

```diff
@@ -0,0 +1,3745 @@
+#!/usr/bin/env python
+# encoding: utf-8
+
+import pytest
+import time
+import re
+import os
+import requests
+import logging
+import tarfile
+import docker
+import json
+import urllib
+import yaml
+import shlex
+import subprocess
+import logging.handlers
+from shutil import copy, copytree, rmtree
+from random import choice
+from string import ascii_lowercase
+from executor import Executor
+from docker.types import Mount
+# Code to suppress insecure https warnings
+import urllib3
+from urllib3.exceptions import InsecureRequestWarning, SubjectAltNameWarning
+urllib3.disable_warnings(InsecureRequestWarning)
+urllib3.disable_warnings(SubjectAltNameWarning)
+
+
+global PLATFORM
+PLATFORM = "debian-9"
+OLD_SPLUNK_VERSION = "7.3.4"
+FILE_DIR = os.path.dirname(os.path.realpath(__file__))
+FIXTURES_DIR = os.path.join(FILE_DIR, "fixtures")
+EXAMPLE_APP = os.path.join(FIXTURES_DIR, "splunk_app_example")
+EXAMPLE_APP_TGZ = os.path.join(FIXTURES_DIR, "splunk_app_example.tgz")
+SCENARIOS_DIR = os.path.join(FILE_DIR, "..", "test_scenarios")
+DEFAULTS_DIR = os.path.join(SCENARIOS_DIR, "defaults")
```

Comment:
You may want to define these top-level variables in `executor.py` so if we create more test scripts, they can leverage this as well.